docs/src/pages/customization/theming/theming.md

```diff
@@ -251,3 +251,38 @@ function App() {
   );
 }
 ```
+
+### `ThemeProvider`
+
+This component takes a `theme` prop, and makes it available down the React tree thanks to the context.
```

Comment:
```suggestion
This component takes a `theme` prop and applies it to the entire React tree that it is wrapping around.
```

I suggest this to point out that `ThemeProvider` applies the received `theme` only to what it's wrapping around.